pulp_ansible/app/utils.py

```diff
@@ -26,3 +27,24 @@ def filter_content_for_repo_version(qs, repo_version):
     content_rel = RepositoryContent.objects.filter(f)
 
     return qs.filter(pk__in=content_rel.values_list("content_id"))
+
+
+def get_queryset_annotated_with_last_sync_task(qs):
+    last_task = (
+        Task.objects.filter(
+            name__contains="sync",
+            reserved_resources_record__icontains=Cast(OuterRef("pk"), output_field=CharField()),
```

Comment:
This assumes that the uuids are truly _universally_ unique. Well...
Can you leave a comment about that?

How bad would it be if the task wasn't found, a wrong one displayed?

Reply:
if the task wasn't found, null is returned. And with `pk`, `name="sync"` and `order_by`, I think there's a little chance that the wrong task would be displayed.